package.json

```diff
@@ -1,11 +1,27 @@
 {
-  "name": "Cuttle",
-  "private": true,
+  "name": "cuttle",
```

Comment:
This goes against the `package.json` naming convention for the `name` property, so I lowercased it while I was in there.